benchmark/main.cpp

```diff
@@ -179,6 +179,8 @@
         mbedtls_printf( "FAILED: -0x%04x\r\n", -ret );
 #endif
 
+#if defined(CoreDebug_DEMCR_TRCENA_Msk) && defined(DWT_CTRL_CYCCNTENA_Msk)
+/* DWT cycle counter is available (Cortex-M3, Cortex-M4) */
```

Comment:
https://github.com/ARMmbed/mbed-os-example-tls/pull/79 removes the cycle counter altogether.